lib/companies_web/views/company_view.ex

```diff
@@ -28,4 +28,10 @@ defmodule CompaniesWeb.CompanyView do
   def select_industries(industries) do
     Enum.map(industries, fn %{id: id, name: name} -> {name, id} end)
   end
+
+  def selected?(%{search: %{"industry_id" => iid}}, industry_id) do
```

Comment:
Can we simplify this into two function heads?


`def selected?(%{search: %{industry_id: industry_id}, industry_id)`

Reply:
Is this function is actually being used?

Functions ending in a question mark also return boolean values, by convention. See https://hexdocs.pm/elixir/naming-conventions.html#trailing-question-mark-foo.

Reply:
@michaelstalker You are right, I used this function when I tried to repopulate the form with the filters already selected, but then since I couldn't make it work, I removed the use of it.

Reply:
Hooray for @michaelstalker 🎉 

Hope you're well good buddy 😁